.github/workflows/deployment-releases.yml

```diff
@@ -0,0 +1,89 @@
+# Deployment workflow to be used for Jira reporting
+name: Deploy releases
+
+on:
+  release:
+    types: [ published ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy-release:
+    name: Deploy a release
+
+    # IMPORTANT: the workflow must have write access to deployments, otherwise the action will fail.
+    permissions:
+      deployments: write
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match-alpha-release
+        with:
+          text: ${{ github.event.release.name }}
```

Comment:
Does this work if the workflow is manually triggered? Because then the event won't be release but workflow_dispatch, right?

Reply:
changed that one  to use github ref_name